src/catering-menus.html

```diff
@@ -0,0 +1,417 @@
+{% markdown %}
+- Sliced ham, turkey, roast beef & cheese with assorted bread
+- Served with lettuce, tomato, onion, pickles
+- Potato Salad
+- Cole Slaw
+- Homemade Cookies
+- Dessert Bars
+- Sweet Tea
+- Unsweet Tea
+- Water
+{% endmarkdown %}
+        </div>
+      </section>
+    </div>
+  </div>
+</div>
+
+<div id="monday" class="block-container">
+  <div class="wrapper">
+    <div class="grid-1-2">
+      <section>
+        <hgroup class="mb-6">
+          <h2 id="monday-breakfast" class="mb-4 text-xl font-bold font-heading lg:text-3xl">
+            Monday - Breakfast
+          </h2>
+          <section class="p-4 bg-blue-100 rounded">
+            <div class="prose prose-lg">
+              <p>
+                There will be dietary restricted meals catered from
+                <a href="https://mediterraneandeli.com/chapel-hill" target="_blank">Mediterranean Deli</a> for those who need them.
+              </p>
+            </div>
+          </section>
+        </hgroup>
+
+        <div class="prose lg:prose-lg">
+{% markdown %}
+- Roasted Country Potatoes
+- Applewood Smoked Bacon
+- Biscuits
+- Fresh Seasonal Fruit
+- Waffles
+- Coffee
+- Orange Juice
+- Water
+{% endmarkdown %}
+        </div>
+      </section>
+      <section>
+        <hgroup class="mb-6">
+          <h2 id="monday-morning-break" class="mb-4 text-xl font-bold font-heading lg:text-3xl">
+            Monday - Morning Break
+          </h2>
+        </hgroup>
+
+        <div class="prose lg:prose-lg">
+{% markdown %}
+- Assorted Dried Fruit
+- Assorted Nuts (Pecans and Almonds)
+- Sunflower Seeds
+- Granola
+- White Chocolate Chips
+- Dark Chocolate Chips
+- M&M's
+- Assorted Soda
+- Bottled Water
+{% endmarkdown %}
+        </div>
+      </section>
+      <section>
+        <hgroup class="mb-6">
+          <h2 id="monday-lunch" class="mb-4 text-xl font-bold font-heading lg:text-3xl">
+            Monday - Lunch
+          </h2>
+          <section class="p-4 bg-blue-100 rounded">
+            <div class="prose prose-lg">
+              <p>
+                There will be dietary restricted meals catered from
+                <a href="https://mediterraneandeli.com/chapel-hill" target="_blank">Mediterranean Deli</a> for those who need them.
+              </p>
+            </div>
+          </section>
+        </hgroup>
+
+        <div class="prose lg:prose-lg">
+{% markdown %}
+- Fried Chicken
+- Chopped Eastern NC Pork BBQ
+- Shrimp and Grits
+- Cornbread
+- Mixed Green Garden
+- Potato Salad
+- Collard Greens
+- Green Beans
+- Apple Cobbler
+- Sweet Tea
+- Unsweet Tea
+- Water
+{% endmarkdown %}
+        </div>
+      </section>
+      <section>
+        <hgroup class="mb-6">
+          <h2 id="monday-afternoon-break" class="mb-4 text-xl font-bold font-heading lg:text-3xl">
+            Monday - Afternoon Break
+          </h2>
+        </hgroup>
+
+        <div class="prose lg:prose-lg">
+{% markdown %}
+- Assorted Ice Cream Novelties
+- Peanuts
+- Crackerjacks
```

Comment:
Isn't cracker jack two words?

Reply:
I copy pasted from the contract, then reordered for the highlights to be first. I'm good with any changes you'd like to suggest.